Dynatrace/dynatrace-operator
# Add OneAgent conditions to the dynakube status

## Description

Add a couple of conditions related to the OneAgent DaemonSet creation:
- [OneAgentVersion conditions](https://github.com/Dynatrace/dynatrace-operator/commit/2b682737a511f7359fe4dca149887e02692c384b)
- [OneAgentConnectionInfo conditions](https://github.com/Dynatrace/dynatrace-operator/commit/4de78721f9e75c22c5c5baa6f527d7b383f665f1)
- [OneAgentDaemonSet conditions](https://github.com/Dynatrace/dynatrace-operator/commit/49ded7e28f9472c2b066a41f73f56512fdd642d3)

## How can this be tested?

Deploy a dynakube which will create a OneAgent DaemonSet, then check the conditions on the DynaKube

## Checklist
